src/main/java/edu/ucla/library/prl/harvester/Op.java

```diff
@@ -4,17 +4,16 @@
 /**
  * OpenAPI operation IDs.
  */
-public final class Op {
+@SuppressWarnings("PMD.FieldNamingConventions")
```

Comment:
What your proposing here is a rule change for enum naming rather than exception to the rules. I think this should involve a change to the PMD rules so that enums are handled differently instead of treated as exceptions: https://pmd.github.io/latest/pmd_rules_java_codestyle.html#fieldnamingconventions

Reply:
The way I think of it is more of an exception though; I've added the warning suppression to enums of identifiers from the OpenAPI definition (Op, Param), but otherwise not (HarvestScheduleStoreService.Error).

Reply:
Although I guess it's arguably become "the rule", since the majority of enum values now fall into that category.

Reply:
Yes, the majority do, I think. And the rule could be loosened to just "any text" so that Error and Op are accepted without exceptions needing to be made.

Reply:
Ok, sounds good to me.